k8s/releases/telegraf/telegraf.yaml

```diff
@@ -0,0 +1,34 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: telegraf
+  namespace: telegraf-stage
+  labels:
+    app: telegraf
+spec:
+  releaseName: telegraf
+  chart:
+    repository: https://helm.influxdata.com/
+    name: telegraf
+    version: "1.8.4"
+  values:
+    envFromSecret: telegraf-influx
+    service:
+      enabled: false
+    config:
+      global_tags:
+        cluster: itse-apps-stage-1-infra
+      inputs:
+        - x509_cert: 
+            sources: 
```

Comment:
since I assume this may create questions.  The intent is to allow us to sample a few websites, and write some medium/low alerts to test the stage deployment itself.  Choosing two highly stable websites as synthetic checks of telegraf.

Reply:
do we want a sample endpoint that we can control / intentionally knock over the certs for, also for testing?

Reply:
That's a cool idea @cmharlow, maybe something we could setup down the line that we could use for multiple things

Reply:
feel free to use something in stage refractr or another staged service for this, fwiw. e.g. paste.stage.mozit.cloud since we can knock over the cert via secret delete (and easily recreate via cert-manager) without little pain to any devs (since we are the devs of it)